### PR TITLE
Implemented logic to display toast for success/error after upload act…

### PR DESCRIPTION
**Pull request by disha1202**
…ion(#2335ca9)

### CODE REVIEW

File: src/views/PurchaseOrder.vue [hunk 1]
```diff
@@ -65,6 +65,8 @@ import { defineComponent } from "vue";
 import { useRouter } from 'vue-router';
 import { parseCsv } from '@/utils';
 import { useStore } from "vuex";
+import { showToast } from '@/utils';
```
Comment (ymaheshwari1) [inline src/views/PurchaseOrder.vue:68]:
```suggestion
import { showToast, parseCsv } from '@/utils';
```
and we can remove line 66.

File: src/views/PurchaseOrder.vue [hunk 2]
```diff
@@ -96,6 +98,12 @@ export default defineComponent({
     methods: {
       getFile(event) {
         this.file = event.target.files[0];
+        if(this.file){
+          showToast(translate("File uploaded successfully"));
+        }
+        else {
+          showToast(translate("Something went wrong, Please try again"));
```
Comment (ymaheshwari1) [inline src/views/PurchaseOrder.vue:105]: Add static text in translation file.

File: src/views/PurchaseOrder.vue [hunk 3]
```diff
@@ -96,6 +98,12 @@ export default defineComponent({
     methods: {
       getFile(event) {
         this.file = event.target.files[0];
+        if(this.file){
+          showToast(translate("File uploaded successfully"));
```
Comment (ymaheshwari1) [inline src/views/PurchaseOrder.vue:102]: We should move `this.parseFile()` inside if, as we only want to parse the file when we have it, otherwise not.

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/views/PurchaseOrder.vue (hunk 1 above)
Comment (disha1202) [reply src/views/PurchaseOrder.vue]: Made the requested change.

File: src/views/PurchaseOrder.vue (hunk 2 above)
Comment (disha1202) [reply src/views/PurchaseOrder.vue:105]: Added static text in translation file.

File: src/views/PurchaseOrder.vue (hunk 3 above)
Comment (disha1202) [reply src/views/PurchaseOrder.vue:101]: Updated the code.

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**